### PR TITLE
Fix mangled comment newlines in code blocks 

### PR DESCRIPTION
**Pull request by wvffle**

### CODE REVIEW

**Comment by wvffle** [comment]
Oh, I've added gocode and gotags there...

File: ui/messages/html/parser.go [hunk 1]
```diff
@@ -276,21 +288,34 @@ func (parser *htmlParser) syntaxHighlight(text, language string) Entity {
 	style := styles.SolarizedDark
 
 	tokens := iter.Tokens()
-	children := make([]Entity, len(tokens))
-	for i, token := range tokens {
+
+	var children []Entity
+	for _, token := range tokens {
+		debug.Print("'" + token.Value + "|" + token.Type.String() + "'")
```
Comment (tulir) [inline ui/messages/html/parser.go:294]:
```suggestion
```

Comment (wvffle) [reply ui/messages/html/parser.go:294]: Good catch

**Review by tulir — APPROVED** [review]
Seems to work, thanks!

**Pull request merged**